syncd/SaiDiscovery.cpp

```diff
@@ -60,6 +60,13 @@ void SaiDiscovery::discover(
 
     sai_object_type_t ot = m_sai->objectTypeQuery(rid);
 
+#ifdef SKIP_SAI_PORT_DISCOVERY
```

Comment:
Just small wu3stion, this check should be done at line 90 after putting port o jest id to discovered set

Reply:
Fixed